### PR TITLE
WRN-1387: Migrate WizardPanels unit tests to Testing Library

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated unit tests from Enzyme to Testing Library for WizardPanels component.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-1387

### CODE REVIEW

**Review by alexandrumorariu — APPROVED** [review]
LGTM

File: WizardPanels/tests/WizardPanels-specs.js [hunk 1]
```diff
@@ -33,48 +27,39 @@ describe('WizardPanel Specs', () => {
...
 
-			const headerTitle = wizardPanel.find('Header').prop('title');
-
-			const expected = viewTitle;
-			const actual = headerTitle;
+			const headerTitle = screen.getByLabelText('step 1 View title undefined');
 
-			wizardPanel.unmount();
-			expect(actual).toBe(expected);
+			expect(headerTitle).toHaveClass('wizard');
```
Comment (daniel-stoian-lgp) [inline WizardPanels/tests/WizardPanels-specs.js:38]: the scope of this test was changed and is not checking the value of the title.

File: WizardPanels/tests/WizardPanels-specs.js [hunk 2]
```diff
@@ -668,18 +678,19 @@ describe('WizardPanel Specs', () => {
...
 			);
 
-			const expected = {total};
-			const actual = wizardPanel.find('Steps').props();
+			const steps = screen.getByText('1' && '2' && '3');
 
-			expect(actual).toMatchObject(expected);
+			await waitFor(() => {
+				expect(steps).toBeInTheDocument();
+			});
```
Comment (daniel-stoian-lgp) [inline WizardPanels/tests/WizardPanels-specs.js:685]: Suggestion:
			
const actual = screen.getByRole('list').children.length;

			await waitFor(() => {
				expect(actual).toBe(total);
			});

File: WizardPanels/tests/WizardPanels-specs.js [hunk 3]
```diff
@@ -647,18 +656,19 @@ describe('WizardPanel Specs', () => {
...
 			);
 
-			const expected = {total: 5};
-			const actual = wizardPanel.find('Steps').props();
+			const steps = screen.getByText('1' && '2' && '3' && '4' && '5');
 
-			expect(actual).toMatchObject(expected);
+			await waitFor(() => {
+				expect(steps).toBeInTheDocument();
+			});
```
Comment (daniel-stoian-lgp) [inline WizardPanels/tests/WizardPanels-specs.js:663]: Suggestion:

			const expected = 5;
			const actual = screen.getByRole('list').children.length;

			await waitFor(() => {
				expect(actual).toBe(expected);
			});

**Review by daniel-stoian-lgp — COMMENTED** [review]
a few more things to solve

File: WizardPanels/tests/WizardPanels-specs.js [hunk 4]
```diff
@@ -85,293 +70,329 @@ describe('WizardPanel Specs', () => {
...
+		'should not hide next button on the last view when `nextButtonVisibility` prop is set to "always"',
 		() => {
-			const wizardPanel = mount(
-				<WizardPanels index={2}>
+			render(
+				<WizardPanels index={2} nextButtonVisibility="always">
 					<Panel>I gots contents</Panel>
 					<Panel>I gots contents2</Panel>
-					<Panel nextButton>Last</Panel>
+					<Panel>Last</Panel>
```
[the remark this answers is not in the document]
Comment (stanca-pop-lgp) [reply WizardPanels/tests/WizardPanels-specs.js:112]: It was changed at some point in the past, but afterwards it was not needed. Thank you for noticing, I undid the changes.

**Review by daniel-stoian-lgp — APPROVED** [review]

**Pull request merged**